submissions/drewzag/task_js_dom/index.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const getData = async () =>
+    fetch('/src/data.json')
+      .then((res) => {
+        if (res.ok) return res.json()
+        else throw new Error('Something went wrong')
+      })
+      .catch((error) => {
+        console.log(error)
+      })
+
+  const list = document.querySelector('.list')
+  const main = document.querySelector('.main')
+  const h1 = document.querySelector('h1')
+  const menuIcon = document.querySelector('div.menu__icon')
+
+  getData().then((res) => {
+    res.forEach((el) => {
+      const li = document.createElement('li')
+      li.classList.add('list__item')
+      li.textContent = el.team
+      list.appendChild(li)
```

Comment:
Adding elements to DOM from a loop is a bad practice. A browser will run reflow and repaint for every element in the loop. Instead, you can:

1. Use append method, which can add several elements in one operation.
2. Create some wrapper, add your items to the wrapper and then add it to DOM. It will be one operation.
3. Clone current container. Add items to a container and then replace your old container with a new one. But be aware of event listeners.
4. Use innerHTML instead.